test/units/module_utils/facts/hardware/linux_data.py

```diff
@@ -484,65 +484,33 @@
         'cpuinfo': open(os.path.join(os.path.dirname(__file__), '../fixtures/cpuinfo/ppc64-power7-rhel7-8cpu-cpuinfo')).readlines(),
         'architecture': 'ppc64',
         'expected_result': {
-            'processor': [
-                '0', 'POWER7 (architected), altivec supported',
-                '1', 'POWER7 (architected), altivec supported',
-                '2', 'POWER7 (architected), altivec supported',
-                '3', 'POWER7 (architected), altivec supported',
-                '4', 'POWER7 (architected), altivec supported',
-                '5', 'POWER7 (architected), altivec supported',
-                '6', 'POWER7 (architected), altivec supported',
-                '7', 'POWER7 (architected), altivec supported'
-            ],
+            'processor': ['0', '1', '2', '3', '4', '5', '6', '7'],
```

Comment:
Changing these fixtures is incorrect. I get test failures just having a list of integers since the output is:

```
cpu_facts['processor']=[
    '0', 'POWER7 (architected), altivec supported', 
    '1', 'POWER7 (architected), altivec supported', 
    '2', 'POWER7 (architected), altivec supported', 
    '3', 'POWER7 (architected), altivec supported', 
    '4', 'POWER7 (architected), altivec supported', 
    '5', 'POWER7 (architected), altivec supported', 
    '6', 'POWER7 (architected), altivec supported', 
    '7', 'POWER7 (architected), altivec supported'
]
```

Reply:
Did you get those outputs with my change or with your suggestion?

Reply:
This was with my suggestion. That would explain the difference.